src/main/java/com/linkedin/replica/wall/models/Comment.java

```diff
@@ -1,4 +1,106 @@
-package com.linkedin.replica.wall.models;
-
-public class Comment {
-}
+package com.linkedin.replica.wall.models;
+
+public class Comment {
+
+        private int commentId,authorId,parentPostId,likesCount,repliesCount;
+        private String [] images;
+        private String [] urls;
+        private int [] mentions;

```

Comment:
Mentions should be an array of String. This should be change in the web service too.